OpenSim/Actuators/Millard2012AccelerationMuscle.h

```diff
@@ -51,7 +51,24 @@ The parent class, Muscle.h, provides
 
 namespace OpenSim {
 
+//==============================================================================
+//                   Millard2012AccelerationMuscle Exceptions
+//==============================================================================
+class Millard2012AccelerationMuscleInvalidFlag : public Exception {
```

Comment:
this and the `Thelen2003MuscleInvalidFlag`  are identical (and subtlly different from `Millard2012EquilibriumMuscleInvalidFlag`), which indicates that they should have a common definition.  I am also not sure of the difference between these exceptions and the general `MuscleCannotEquilibrate` since they will be thrown in `computeInitialFiberEquilibrium` in which case the *invalid flag* context seems more mystifying than informative. There is no added information for what causes the invalid flag and the user is being told about an internal private method they have no control over.

My suggestion is to enable the general `MuscleCannotEquilibrate` to take additional details (string) about why the failure occurred. If more information can be added by the failure of internal helper methods then that could be tacked on as details. I would be satisfied if the specific contents of the "details" are a TODO (simply stating the internal method that failed) for when the initialization/equilibrium methods are revisited. At least then you get the correct top level error/message that the MuscleCannotEquilibrate and you get the internal method as a detail rather than a mysterious `###MuscleInvalidFlag` exception.

Please chat with me if this is unclear or seems unnecessary.